File: python_ruby workspace/class_member_2.py
class Cs :
    count = 0
    def __init__(self):
        Cs.count = Cs.count + 1
    @classmethod
    def getCount(cls):
        return Cs.count

i1 = Cs()
i2 = Cs()
i3 = Cs()
i4 = Cs()

print(Cs.getCount())
#몇개의 인스턴스가 생성됐는지 세는 메소드 정의

File: python_ruby workspace/class_member_2.rb
class Cs
    @@count = 0
    def initialize()
        # @count 인스턴스에 속함
        @@count = @@count + 1 #클래스에 속함
    end
    def Cs.getCount()
        return @@count  
    end
end


i1 = Cs.new()
i2 = Cs.new()
i3 = Cs.new()
i4 = Cs.new()

p Cs.getCount()
#몇개의 인스턴스가 생성됐는지 세는 메소드 정의